src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -91,9 +91,12 @@ const filterConfigIO = ({ inputs, outputs }) => {
   )
 }
 
-const WorkflowIOTable = ({ which, inputsOutputs: data, config, errors, onChange, onSetDefaults, onBrowse, suggestions, readOnly }) => {
+const WorkflowIOTable = ({ which, inputsOutputs: data, config, errors, onChange, onSetDefaults, onBrowse, suggestions, availableSnapshots, readOnly }) => {
   const [sort, setSort] = useState({ field: 'taskVariable', direction: 'asc' })
 
+  // will only match if the current root entity type comes from a snapshot
+  const isSnapshot = !!_.find({ name: config.dataReferenceName }, availableSnapshots)
```

Comment:
Would `_.some` be a better fit since we don't seem to care about the value `_.find` may return?

```suggestion
  const isSnapshot = _.some({ name: config.dataReferenceName }, availableSnapshots)
```

Reply:
good call, thank you!